crud/common/utils.lua

```diff
@@ -413,6 +413,63 @@ function utils.format_result(rows, space, field_names)
     return result
 end
 
+local function truncate_tuple_metadata(tuple_metadata, field_names)
+    dev_checks('?table', 'table')
+
+    if tuple_metadata == nil then
+        return nil
+    end
+
+    local truncated_metadata = {}
+
+    if #tuple_metadata < #field_names then
+        return nil, FilterFieldsError:new(
+                'Field names don\'t match to tuple metadata'
+        )
+    end
+
+    for i, name in ipairs(field_names) do
+        if tuple_metadata[i].name ~= name then
+            return nil, FilterFieldsError:new(
+                    'Field names don\'t match to tuple metadata'
+            )
+        end
+
+        table.insert(truncated_metadata, tuple_metadata[i])
+    end
+
+    return truncated_metadata
+end
+
+function utils.cut_rows(rows, metadata, field_names, opts)
```

Comment:
If it's public method you should provide some documentation.
Especially if you a refer this part above https://github.com/tarantool/crud/pull/143/files#diff-24b635b35abaadcc2951d808149b57dcb8fe2325d9b08d2c861eaffed77ce167R72-R74